config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -361,3 +361,200 @@ periodics:
       - --test-cmd-name=ClusterLoaderV2
       - --timeout=40m
       - --use-logexporter
+
+# max volumes per cluster test cases for ephemeral volumes
+- name: ci-kubernetes-storage-scalability-max-emptydir-vol-per-cluster-500
+  tags:
+  - "perfDashPrefix: storage-max-emptydir-vol-per-cluster-500"
+  - "perfDashJobType: storage"
+  interval: 1h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+    preset-e2e-kubemark-common: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-experiments
+    testgrid-tab-name: max-emptydir-vol-per-cluster-500
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190719-7cdf877-master
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
+      - --timeout=140
+      - --scenario=kubernetes_e2e
+      - --
+      - --cluster=kubemark-500
+      - --extract=ci/latest
+      - --gcp-master-size=n1-standard-4
+      - --gcp-node-image=gci
+      - --gcp-node-size=n1-standard-8
+      - --gcp-nodes=8
+      - --gcp-project=k8s-jenkins-blocking-kubemark
+      - --gcp-zone=us-central1-f
+      - --kubemark
```

Comment:
The other question is whether kubemark is the setup we really like to test that. I'm actually not fully convinced whether we will not miss anything critical (we're e.g. faking docker in kubemark).
@msau42 - I would like to hear your opinion whether kubemark is fine for ephemeral storage.

Reply:
The container runtime does contribute to pod startup (it does one more bind mount of the volume into the container).

For persistent volumes, kubemark definitely won't work

Reply:
For persistent volumes, I know that.

My question was more about ephemeral ones - do you think the the difference would be small enough that we can accept it?

Reply:
I can't say for sure but overall the runtime bind mounting would contribute to latency at an individual node level and not a cluster wide level.  So I guess the single node stress tests we have should cover any scaling issues with the runtime mounts themselves.

Can we do a manual run with and without kubemark to see the numbers?